appeals/web/src/server/lib/mappers/appeal/has.js

```diff
@@ -0,0 +1,96 @@
+import { mapAppellant } from './submappers/appellant.mapper.js';
+import { mapCaseHistory } from './submappers/case-history.mapper.js';
+import { mapCaseOfficer } from './submappers/case-officer.mapper.js';
+import { mapCaseProcedure } from './submappers/case-procedure.mapper.js';
+import { mapCompleteDate } from './submappers/complete-date.mapper.js';
+import { mapCostsAppellantApplication } from './submappers/costs-appellant-application.mapper.js';
+import { mapCostsAppellantCorrespondence } from './submappers/costs-appellant-correspondence.mapper.js';
+import { mapCostsAppellantWithdrawal } from './submappers/costs-appellant-withdrawal.mapper.js';
+import { mapCostsDecision } from './submappers/costs-decision.mapper.js';
+import { mapCostsLpaApplication } from './submappers/costs-lpa-application.mapper.js';
+import { mapCostsLpaCorrespondence } from './submappers/costs-lpa-correspondence.mapper.js';
+import { mapCostsLpaWithdrawal } from './submappers/costs-lpa-withdrawal.mapper.js';
+import { mapCrossTeamCorrespondence } from './submappers/cross-team-correspondence.mapper.js';
+import { mapDecision } from './submappers/decision.mapper.js';
+import { mapFinalCommentReviewDueDate } from './submappers/final-comment-review-due-date.mapper.js';
+import { mapInspectorCorrespondence } from './submappers/inspector-correspondence.mapper.js';
+import { mapInspectorNeighbouringSites } from './submappers/inspector-neighbouring-sites.mapper.js';
+import { mapInspector } from './submappers/inspector.mapper.js';
+import { mapIssueDeterminationDate } from './submappers/issue-determination-date.mapper.js';
+import { mapLeadOrChild } from './submappers/lead-or-child.mapper.js';
+import { mapLinkedAppeals } from './submappers/linked-appeals.mapper.js';
+import { mapLocalPlanningAuthority } from './submappers/local-planning-authority.mapper.js';
+import { mapLpaHealthAndSafety } from './submappers/lpa-health-and-safety.mapper.js';
+import { mapLpaInspectorAccess } from './submappers/lpa-inspector-access.mapper.js';
+import { mapLpaNeighbouringSites } from './submappers/lpa-neighbouring-sites.mapper.js';
+import { mapLpaQuestionnaireDueDate } from './submappers/lpa-questionnaire-due-date.mapper.js';
+import { mapLpaQuestionnaire } from './submappers/lpa-questionnaire.mapper.js';
+import { mapLpaReference } from './submappers/lpa-reference.mapper.js';
+import { mapNeighboringSiteIsAffected } from './submappers/neighboring-site-is-affected.mapper.js';
+import { mapOtherAppeals } from './submappers/other-appeals.mapper.js';
+import { mapSiteAddress } from './submappers/site-address.mapper.js';
+import { mapSiteVisitDate } from './submappers/site-visit-date.mapper.js';
+import { mapStartedAt } from './submappers/started-at.mapper.js';
+import { mapStatementReviewDueDate } from './submappers/statement-review-due-date.mapper.js';
+import { mapValidAt } from './submappers/valid-at.mapper.js';
+import { mapVisitType } from './submappers/visit-type.mapper.js';
```

Comment:
That's a lot of imports! Maybe an index file of sorts would help here?

Reply:
There'd still be one really big import statement even with an index file, and we'd incur the overhead of keeping the index file up to date. Unless we create the export programatically that is. Maybe an interesting thing we can pair on later in the week?

Reply:
Good point re. maintenance, I'd rather have them explicitly defined rather than generated for the sake of clarity, so happy to leave this one